src/localplayer.h

```diff
@@ -145,39 +105,18 @@ class LocalPlayer : public Player
 	float getZoomFOV() const { return m_zoom_fov; }
 	void setZoomFOV(float zoom_fov) { m_zoom_fov = zoom_fov; }
 
+protected:
+	virtual bool checkPrivilege(const std::string &priv) const;
+	virtual void triggerJumpEvent();
+
+	virtual const NodeDefManager *getNodeDefManager() const;
+	virtual void _handleAttachedMove();
```

Comment:
This function looks suspicious - what's the for the underline?

Reply:
Oops, probably just a leftover from having worked in a language that uses that convention for private stuff. I'll remove it.